src/aleph/http.clj

```diff
@@ -88,6 +88,9 @@
 (def default-response-executor
   (flow/utilization-executor 0.9 256 {:onto? false}))
 
+(def default-timer
```

Comment:
Should this be `defonce`?

Reply:
@ztellman It definitely should! I'm waiting for the feedback from @alexander-yakushev if this even worse being changed. If so, I will complete the implementation and polishing.

Reply:
Done.